zerver/lib/webhooks/common.py

```diff
@@ -32,3 +39,18 @@ def check_send_webhook_message(
             # stream, so we don't need to re-raise it since it clutters up
             # webhook-errors.log
             pass
+
+def validate_extract_webhook_http_header(header: Text, request: HttpRequest,
+                                         user_profile: UserProfile) -> Text:
+    extracted_header = request.META.get(header)
+    if extracted_header is None:
+        message_body = MISSING_EVENT_HEADER_MESSAGE.format(
+            bot_name=user_profile.full_name)
+        internal_send_private_message(
+            user_profile.realm,
+            get_system_bot(settings.NOTIFICATION_BOT),
+            user_profile.bot_owner,
+            message_body
+        )
```

Comment:
Do we want to raise a `JsonableError` here, so that callers don't need to handle the case where this returns `None`?